docsets/Agora_CPP/docset.json

```diff
@@ -1,22 +1,38 @@
 
 {
     "name": "Agora C++",
-    "version": "2.9.0",
+    "version": "3.0.0",
     "archive": "Agora_CPP.tgz",
     "author": {
         "name": "Agora",
         "link": "https://www.agora.io"
     },
     "aliases": [],
     "specific_versions": [
+        {
+            "_comment": "Agora C++ API Reference for All Platforms v2.9.0"
+        },
+        {
+            "version": "2.9.0",
+            "archive": "versions/2.9.0/Agora_CPP.tgz"
+        },
+        {
+            "_comment": "Agora C++ API Reference for All Platforms v2.8.0"
+        },
         {
             "version": "2.8.0",
-            "archive": "versions/2.8.0/Agora_CPP.tgz",
+            "archive": "versions/2.8.0/Agora_CPP.tgz"
```

Comment:
You need to actually add this archive. Renaming/moving the txt file does nothing.

Reply:
Archive fixed.